docs/data/material/design-resources/material-ui-for-figma/material-ui-for-figma.md

```diff
@@ -0,0 +1,116 @@
+# Material UI for Figma
+
+<p class="description">Enhance the designer-developer working with Material UI collaboration.</p>
+
+## Getting started
+
+Material UI for Figma consists of a representation of the React component in Figma so that designers and developers can communicate and iterate more efficiently as they work with a set of seamlessly integrated tools.
+
+### Community vs. full version
+
+The Material UI design kits is available in [community (free)](https://www.figma.com/community/file/912837788133317724/material-ui-for-figma-and-mui-x) and [full (paid)](https://mui.com/store/items/figma-react/) versions.
+Here's what you can expect from each one of them:
+
+|                                   | Community | Full version |
+| :-------------------------------- | --------: | -----------: |
+| Components without customizations |       All |          All |
+| Components with customizations    |         4 |          All |
+| Figma variables                   |         - |           ✅ |
+
+### How to install the full version?
+
+Start by extracting the `.zip` archive containing a couple of `.fig` files.
+Then, you can either follow [Figma's import guide](https://help.figma.com/hc/en-us/articles/360041003114-Import-files-into-Figma) or [add it to your team library](https://help.figma.com/hc/en-us/articles/360041051154-Getting-Started-with-Team-Library).
+
+## Theme
+
+### When I open the local variables panel, I see only 2 values; where are the rest?
+
+When opening the local variables panel for the first time, you will see this:
+
+<!-- ![Screenshot 2023-08-08 at 01.09.47.png](https://s3-us-west-2.amazonaws.com/secure.notion-static.com/4f2dd392-50f8-4b25-99ef-c52815fbc803/Screenshot_2023-08-08_at_01.09.47.png) -->
```

Comment:
I assume this is WIP. It makes a lot of sense to me for @DavidCnoops to take ownership of this PR and the docs in general. Danilo: spread too thin, I: spread too thin, Adrian: outside of the org.